cds_ils/migrator/relations/api.py

```diff
@@ -116,25 +116,30 @@ def migrate_document_siblings_relation():
                     document_class, legacy_pid_type, relation["related_recid"]
                 )
             except PIDDoesNotExistError as e:
-                pass
-
-            # try to find sibling in series
-            if related_sibling is None:
-                try:
-                    related_sibling = get_record_by_legacy_recid(
-                        series_class,
-                        legacy_pid_type,
-                        relation["related_recid"],
+                continue
+            rec_type = related_sibling["$schema"].split("/")[-1].split("-")[0]
```

Comment:
why not checking with `instance of` or `type`?

Reply:
Because it returns a dict, not a Document or Series instance